mmv1/templates/terraform/examples/eventarc_workflows.tf.erb

```diff
@@ -0,0 +1,108 @@
+# [START eventarc_terraform_enableapis]
+# Used to retrieve project_number later
+data "google_project" "project" {
+  provider = google-beta
+}
+
+# Enable Eventarc API
+resource "google_project_service" "eventarc" {
+  provider = google-beta
+  service            = "eventarc.googleapis.com"
+  disable_on_destroy = false
+}
+
+# Enable Pub/Sub API
+resource "google_project_service" "pubsub" {
+  provider = google-beta
+  service            = "pubsub.googleapis.com"
+  disable_on_destroy = false
+}
+
+# Enable Workflows API
+resource "google_project_service" "workflows" {
+  provider = google-beta
+  service            = "workflows.googleapis.com"
+  disable_on_destroy = false
+}
+
+# [END eventarc_terraform_enableapis]
+
+# [START eventarc_workflows_create_serviceaccount]
+
+# Create a service account for Eventarc trigger and Workflows
+resource "google_service_account" "eventarc_workflows_service_account" {
+  provider = google-beta
+  account_id   = "eventarc-workflows-sa"
+  display_name = "Eventarc Workflows Service Account"
+}
+
+# Grant the logWriter role to the service account
+resource "google_project_iam_binding" "project_binding_eventarc" {
+  provider = google-beta
+  project = data.google_project.project.id
+  role    = "roles/logging.logWriter"
+
+members = [
```

Comment:
nit: would you mind lining up all these `members` lists to line up with the other attributes and remove the extra whitespace? thanks!